benchmarking/nursery/benchmark_automl/baselines.py

```diff
@@ -32,14 +32,17 @@
 
 @dataclass
 class MethodArguments:
-    config_space: Dict
+    config_space: dict
```

Comment:
This changes seems unintended since Dict is the type.

Reply:
I am general trying to change Dict to dict when nothing else is said, like Dict[str, int] or so. Same for List -> list, or Tuple -> tuple.

Reply:
Makes sense.